docs/tasks/job.md

```diff
@@ -163,6 +165,8 @@ This is an example playbook that creates a new Protection Job for the chosen vCe
                 sources: 
                   - endpoint: "{{ var_endpoint }}"
                 environment: "VMware"
+                time_zone: "Asia/Kolkata"
+                start_time: "12:34"
```

Comment:
If we are accepting military time, better give example like 21:34

Reply:
Also best way is to give a list of acceptable timezone names such as, can you find out where can we get this list . Start here: https://docs.python.org/3/library/datetime.html#timezone-objects, This would help in our Python SDK as well.

Reply:
We can get the timezones using pytz module.
Where can we add the list.

Reply:
Sure,  instead of providing entire list , give an example how they can select a timezone with pytz , but this is ansible module , perhaps just make a comment.